docs/source/notebooks/finch-usage.ipynb

```diff
@@ -64,7 +64,7 @@
       "    Parameters\n",
       "    ----------\n",
       "    tasmin : ComplexData:mimetype:`application/x-netcdf`, :mimetype:`application/x-ogc-dods`\n",
-      "        NetCDF Files or archive (tar/zip) containing netCDF files.\n",
+      "        NetCDF Files or archive (tar/zip) containing netCDF files. Minimum surface temperature.\n",
```

Comment:
This "Minimum surface temperature." text addition breaks Jenkins :(  New Finch 0.7.5 is good to be deployed to PAVICS?